docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -67,9 +67,9 @@ There are three levels of configuration:
 |The Kafka version used in the format: x.y.z
 |
 
-|zookeeperReference
-|struct
-|A reference to a ZooKeeper cluster (namespace, name, chroot)
+|zookeeperConfigMapName
+|string
+|A reference to a ConfigMap containing a connection string for connecting to a ZooKeeper cluster
```

Comment:
I can only comment here: Above (line 12-39) is still an old custom resource example.

Reply:
Good catch, updated.